### PR TITLE
Rex jtd main config local

### PR DESCRIPTION
**Pull request by JonTri-5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,13 +79,33 @@ Update required configuration variables located in `readml/config/config_local.c
...
+You may update required configuration variables located in `readml/config/config_local.cfg` before.
 
 For instance , using **partial dependency plots** for global interpretability on ML model:
 ```python
 from readml.main import interpret
 interpret(interpret_type="global", use_pdp_ice=True, use_ale=False, use_shap=False)
 ```
+
+In the other way, you can give as an input a dictionnary into `interpret` function with the same config keys located in `readml/config/`, as an example, after the run of pytest :
+
```
Comment (soumayaihihi) [inline README.md:92]: Proposition de reformulation :

You can also use the `interpret` function without filling in the configuration file. To do so, you need to pass a config dictionary, as additional parameter to the function, with the same config keys located in `readml/config/` (you will find templates for each usecase). 
Here is an example using the outputs of unit tests :

Comment (soumayaihihi) [inline README.md:92]: Proposition de reformulation :

You can also use the `interpret` function without filling in the configuration file. To do so, you need to pass a config dictionary, as additional parameter to the function, with the same config keys located in `readml/config/` (you will find templates for each usecase). 
Here is an example using the outputs of unit tests :

File: readml/main.py [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 from readml.utils import _parse_and_check_config
 
 
 @click.command()
+@click.option(
+    "--config-values",
+    default=None,
+    show_default=True,
+    metavar="",
+    help="Dictionnary of configuration adapted to the type of problematic you need to interpret",
```
Comment (soumayaihihi) [inline readml/main.py]: Add that templates are available in `readml/config/` folder

Comment (soumayaihihi) [inline readml/main.py]: Add that templates are available in `readml/config/` folder

File: readml/main.py [hunk 3]
```diff
@@ -91,4 +105,7 @@ def interpret(
 
 
 if __name__ == "__main__":
+    # After the use of pytest, you can run :
+    # For tabular ML : python3 readml/main.py --config-values='{"model_path": "/workspaces/readml/outputs/tests/core/model/model.sav" ,"out_path": "/workspaces/readml/outputs/tests/core/","task_name": "regression","learning_type": "ML","data_type": "tabular","features_to_interpret": "F1,F2","tree_based_model": "True","features_name": "F1,F2,F3","target_col": "target","train_data_path": "/workspaces/readml/outputs/tests/core/data/train.csv","train_data_format": "csv","test_data_path": "/workspaces/readml/outputs/tests/core/data/train.csv","test_data_format": "csv"}'
```
Comment (soumayaihihi) [inline readml/main.py:109]: Why add this here?

Comment (soumayaihihi) [inline readml/main.py:109]: Why add this here?

File: README.md [hunk 4]
```diff
@@ -187,6 +207,10 @@ python readml/main.py --help
 Usage: main.py [OPTIONS]
 
 Options:
+  --config-values    Dictionnary of configuration adapted to the type of
```
Comment (soumayaihihi) [inline README.md:211]: Same as below 
Add that templates are available in readml/config/ folder

File: readml/main.py (hunk 3 above)
Comment (JonTri-5) [reply readml/main.py:109]: This commentaries are here to help futur developer to use the main.py on an example automatically generated with pytest.

Maybe this is not the place to conserve it ? Must I drop it or just move it ?

File: README.md (hunk 1 above)
Comment (JonTri-5) [reply README.md:92]: Ok, for your proposition.
Done :)

File: README.md (hunk 4 above)
Comment (JonTri-5) [reply README.md:211]: Done :)

File: readml/main.py (hunk 2 above)
Comment (JonTri-5) [reply readml/main.py]: Done :)

File: readml/main.py (hunk 3 above)
Comment (soumayaihihi) [reply readml/main.py:109]: If you want to run a quick test of the CLI command without needing data or model, you can use unit tests outputs

**Pull request merged**